### ISSUE TITLE
Errors in Visual Studio Code while opening a brand new Uno Platform solution

### ISSUE THREAD

**Issue by mobynet1** [unoplatform/uno#17732]
### Current behavior

I created a new Uno Platform solution using the online wizard. When I loaded the solution for the first time, I received the same error which was repeated four times in the output window in Visual Studio code: `Error while loading xxx.csproj: Project '..\xxx.csproj' targets 'net8.0-android;net8.0-ios;net8.0-maccatalyst;net8.0-browserwasm;net8.0-desktop;net8.0-windows10.0.19041.0'. It cannot be referenced by a project that targets '.NETCoreApp,Version=v8.0'.` I detest seeing errors or warnings of any kind in a solution so I deleted this solution, opened Visual Studio 2022 and created a fresh Uno solution. When I did a solution clean build no issues were reported so I opened the same solution in Visual Studio Code and saw these errors.

I am watching VS Code right now and it is in an infinite loop trying to load the solution.

```
[Error - 8:29:13 AM] [LanguageServerProjectSystem] Error while loading c:\Development\MobyNet\Dev-Uno\MobyClient\MobyClient\MobyClient.csproj: Project MobyClient.MauiControls is not compatible with net8.0-desktop1.0 (.NETCoreApp,Version=v8.0) / win-x64. Project MobyClient.MauiControls supports:
  - net8.0-android34.0 (.NETCoreApp,Version=v8.0)
  - net8.0-ios17.2 (.NETCoreApp,Version=v8.0)
  - net8.0-maccatalyst17.2 (.NETCoreApp,Version=v8.0)
  - net8.0-windows10.0.19041 (.NETCoreApp,Version=v8.0)
```

just keeps getting repeated in the output window.

Update: I deleted that solution and created a new solution using Visual Studio 2022 and only included the bare minimum options. It runs, but I get errors in files that I should not be getting with a new, out-ot-the-box solution. As much as I detest errors in log files or output windows, I equally detest the red squigglies. I opend Presentations\Shell.xaml.cs and see that it is unbound to the Shell.xaml file - `'App' does not contain a definition for 'InitializeComponent' and no accessible extension method 'InitializeComponent' accepting a first argument of type 'App' could be found (are you missing a using directive or an assembly reference?)`

### Expected behavior

No Errors! Obviously! I truly get annoyed when I download a sample or something from GitHub and it does not compile cleanly. You guys need to check this out and get it cleaned up!! What I know of Uno excites me, but my enthusiasm took a hit by just trying to get a CLEAN install.

### How to reproduce it (as minimally and precisely as possible)

I said everything necessary in the opening statement.

### Workaround

Haven't found one yet...

### Works on UWP/WinUI

None

### Environment

Uno.UI / Uno.UI.WebAssembly / Uno.UI.Skia, Uno.WinUI / Uno.WinUI.WebAssembly / Uno.WinUI.Skia, Uno.SourceGenerationTasks, Uno.UI.DevServer / Uno.WinUI.DevServer

### NuGet package version(s)

This is a freshly created solution. You should be providing everything a developer needs in this initial creation.

### Affected platforms

WebAssembly, Android, iOS, Mac Catalyst, Skia (WPF), Skia (Linux X11), Skia (macOS), Skia (Linux Framebuffer), Skia (GTK), Windows (WinAppSDK), Build tasks

### IDE

Visual Studio 2022, Visual Studio Code

### IDE version

VS Code 1.91.1; Visual Studio 2022 Version 17.10.4

### Relevant plugins

Uno Platform extension for both VS versions
Figma plug-in

### Anything else we need to know?

I hope this is not a harbinger of things to come............ I am in the process of pulling the plug on .NET Maui because it does not deliver. The fact that I cannot get a clean install is worrisome.

**Comment by jeromelaban** [comment]
Thanks for the report. Can you provide the Wizard-generated command line you used to create the app that got you this error? 

Note that C# errors coming up in VS Code are tied to support from C# Dev Kit, which is Microsoft-provided. In the case you're looking at, it may very well be that the code generation tooling from WinUI does not run, and that may go away once a build has succeeded. If you do not include the `Windows` platform in the Wizard, do you still have the `InitializeComponent` error? If so, we'll track this on the WinUI side.

**Comment by mobynet1** [comment]
Hi @jeromelaban, thanks for the response. I tried to use the online wizard (because I wanted to use VSC because it supports Figma) and also through Visual Studio 2022 Project->New which also triggers the wizard. The string from the wizard I used is long gone so I do not recall what it was exactly.

I know the red squiggly error in App.xaml/App.xaml.cs is not your issue because I have had it happen on numerous occasions with Maui. I do know that this disconnect happens after the csproj file is manually edited. I am suspecting the encoding of the file itself which has bounced back and forth between 'lf', 'crlf', and 'mixed'. When I try to reproduce this issue, I can't find the right circumstances to get it to act up. I had been making edits to the project file to specify changes I was trying for the app icon and for the splash screen. dunnoh...
